core/main/src/broker/thunder/thunder_plugins_status_mgr.rs

```diff
@@ -416,19 +416,17 @@ impl StatusManager {
             }
             self.update_status(callsign.to_string(), status.to_state());
 
-            if status.to_state().is_activated() {
-                let (pending_requests, expired) =
-                    self.retrive_pending_broker_requests(callsign.to_string());
-
-                for pending_request in pending_requests {
-                    if expired {
-                        info!("Expired request: {:?}", pending_request);
-                        callback
-                            .send_error(pending_request, RippleError::ServiceError)
-                            .await;
-                    } else {
-                        let _ = sender.send(pending_request).await;
-                    }
+            let (pending_requests, expired) =
+                self.retrive_pending_broker_requests(callsign.to_string());
+
+            for pending_request in pending_requests {
+                if expired {
+                    info!("Expired request: {:?}", pending_request);
```

Comment:
I realize that this is from the original code but: does this need to be info!? maybe debug is better?

Reply:
+1